Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -69,5 +90,10 @@ try {
         Publish-NewServiceFabricApplication -ApplicationPackagePath $ApplicationPackagePath -ApplicationParameterFilePath $publishProfile.ApplicationParameterFile -Action RegisterAndCreate -OverwriteBehavior SameAppTypeAndVersion -ErrorAction Stop 
     }
 } finally {
+    if ($certificate)
+    {
+        Remove-Certificate -ClusterConnectionParameters $clusterConnectionParameters -Certificate $certificate
```

Comment:
Is it possible they don't want to remove the certificate? For example what if the build machine already had the certificate installed and uses it for some other CI process as well? If we remove the cert in that case, we would cause the other CI process to fail.